src/rpc/net.cpp

```diff
@@ -114,7 +114,7 @@ static RPCHelpMan getpeerinfo()
                     {
                         {RPCResult::Type::STR, "SERVICE_NAME", "the service name if it is recognised"}
                     }},
-                    {RPCResult::Type::BOOL, "relaytxes", /*optional=*/true, "Whether peer has asked us to relay transactions to it"},
+                    {RPCResult::Type::BOOL, "relaytxes", "Whether we relay transactions to this peer"},
```

Comment:
in cdad951abf73650a9b49dd038ecbcc96ebf9bc0f: I think the doxygen for `m_relay_txs` should also be updated for the same reason. Otherwise it is not clear why you are changing this.

Reply:
Good idea, done.

Reply:
> done

No?

Reply:
Thanks, pushed